docs/pages/versions/v52.0.0/sdk/maps.mdx

```diff
@@ -88,6 +88,44 @@ Expo Maps provides access to the platform native map APIs on Android and iOS.
 
 </Collapsible>
 
+To display the user's location on the map, you need to declare and request location permission beforehand. You can configure this using the built-in [config plugin](/config-plugins/introduction/) if you use config plugins in your project ([EAS Build](/build/introduction) or `npx expo run:[android|ios]`). The plugin allows you to configure various properties that cannot be set at runtime and require building a new app binary to take effect.
+
+<ConfigPluginExample>
```

Comment:
```suggestion
<ConfigPluginExample>
```

You'll have to import both `import { ConfigPluginExample, ConfigPluginProperties } from '~/ui/components/ConfigSection';` to resolve this and the one below.